pygen/pygen_src/riscv_instr_gen_config.py

```diff
@@ -111,16 +111,16 @@ def __init__(self, argv):
         self.num_of_harts = argv.num_of_harts
         self.fix_sp = argv.fix_sp
         self.use_push_data_section = argv.use_push_data_section
-        self.boot_mode_opts = argv.boot_mode_opts
+        self.boot_mode_opts = argv.boot_mode
```

Comment:
same here, I think the `boot_mode` and `init_privileged_mode` fixes are in #694?

Reply:
edit: resolved above

Reply:
That's true. We will only merge this current PR when it is approved. Previous PR just meant to be small fixes to the pyflow generator, and I wanted to keep it separate so it would be easier for perfectVIP team to review it.